mappings/net/minecraft/network/packet/c2s/play/PlayerInteractEntityC2SPacket.mapping

```diff
@@ -2,5 +2,68 @@ CLASS net/minecraft/class_2824 net/minecraft/network/packet/c2s/play/PlayerInter
 	FIELD field_12870 entityId I
 	FIELD field_12871 type Lnet/minecraft/class_2824$class_5906;
 	FIELD field_25660 playerSneaking Z
```

Comment:
Honestly we should really check what sneak this is. Entity has 4 types of sneaks I recall

Reply:
The one that goes Entity#isSneaking

Reply:
Hmm, then this is the basic sneak. Sounds good :+1: